docker-compose.yml

```
@@ -8,14 +8,15 @@ services:
   #   ports:
   #     - ${APP_PORT}:${APP_PORT}
   db:
-    image: mysql:5.7.30
+    image: mariadb:latest
```

```suggestion
    # image: mariadb:10.5.5
    image: mysql:5.7.30
```
Prefer fixed version to avoid any problem caused by version upgrade

另外main branch保留mysql, 因為有其他人在試
之後另開一條branch用mariadb
ETA: this week end

---

已修正，image 放到 .env 內